<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomMenuButton</class>
 <widget class="QWidget" name="CustomMenuButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>141</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxCustomMenuButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblCustomMenuButton">
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="scopy::gui::CustomPushButton" name="btnCustomMenuButton">
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scopy::gui::CustomPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">scopy/gui/custom_push_button.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
